lib/galaxy/files/sources/s3fs.py

```diff
@@ -28,60 +26,61 @@
 
 DEFAULT_ENFORCE_SYMLINK_SECURITY = True
 DEFAULT_DELETE_ON_REALIZE = False
+FS_PLUGIN_TYPE = "s3fs"
 
 log = logging.getLogger(__name__)
 
 
-class S3FsFilesSourceProperties(FilesSourceProperties, total=False):
-    bucket: str
-    endpoint_url: int
-    user: str
-    passwd: str
-    listings_expiry_time: NotRequired[Optional[int]]
```

Comment:
Was there a reason to drop this ? I think this caused https://github.com/galaxyproject/galaxy/issues/20791 ?